docs/sphinx/source/user_guide/roadmap.rst

```diff
@@ -0,0 +1,153 @@
+.. _roadmap:
+
+pvlib Roadmap
+=============
+
+**Last revision: October 2023**
+
+This document outlines a "wish list" for advancing pvlib-python, focusing
+on gaps in functionality and other code-related areas of improvement.
+
+Achieving these improvements will depend on community contributions.
+Therefore, it's not reasonable to set forth a timeline for this roadmap
+to be completed.
+For a more concrete plan of short-term additions to pvlib, please see
+the `milestone page <https://github.com/pvlib/pvlib-python/milestones>`_
+on GitHub.
+
+Additionally, an idea being listed here is no guarantee that it
+will ever be added to pvlib, nor is omission from this page an
+indication that an idea is not of interest.  This roadmap is an update
+of previous roadmaps based on requests from users:
+
+* `2023 pvlib-python user meeting notes
+  <https://github.com/pvlib/pvlib-python/wiki/2023-pvlib-python-user-meeting-notes>`_
+* `2022 pvlib development roadmap
+  <https://github.com/pvlib/pvlib-python/discussions/1581>`_
+* `2019 pvlib-python development roadmap
+  <https://github.com/pvlib/pvlib-python/wiki/2019-pvlib-python-Development-Roadmap>`_
+
+
+Core modeling capabilities
+--------------------------
+The core functionality of pvlib is its library of model functions,
+organized by the various conceptual steps of a PV system performance
+simulation.  pvlib has robust modeling capability in many of these topic
+areas, but coverage in some areas is poor or nonexistent.  Here is a
+partial list of modeling areas that could be improved:
+
+* Improved inverter performance models (power factor, off-MPP operation,
+  thermal derating, MPPT voltage range)
+* DC-DC optimizer performance models
+* Transformer models
+* Electrical mismatch functionality
+* Improved snow loss models, in particular for bifacial and tracked systems
```

Comment:
IIRC from the May 2023 PVPMC, one of the Sandia presentations talked about multijunction/tandem modelling goals for PVPMC as a whole(?)
Also, on the subject of irradiance, the spectral effect on the calculation effective POA irradiance was highlighted in the github wiki 2023 user meeting notes. I think this area still needs work, especially if/when multijunction modelling functionality is added

```suggestion
* Improved snow loss models, in particular for bifacial and tracked systems
* Multijunction device modelling
```